### PR TITLE
Zero-onboarding recipe updated to match new schema

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

**Comment by AllegroFox** [comment]
I'm still seeing an error when I run `make check` with this JSON, but I'm not sure that my schema is correctly updated. Ignoring errors for the other experiments, for MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST I see:

```
Validate schema for MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST
Validate MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST with Experiments schema 82+
Validation error against the new experiments schema: 'slug' is a required property
Validation error: 'value' is a required property
make: *** [check] Error 1
```

**Comment by punamdahiya** [comment]
> I'm still seeing an error when I run `make check` with this JSON, but I'm not sure that my schema is correctly updated. Ignoring errors for the other experiments, for MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST I see:
> 
> ```
> Validate schema for MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST
> Validate MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST with Experiments schema 82+
> Validation error against the new experiments schema: 'slug' is a required property
> Validation error: 'value' is a required property
> make: *** [check] Error 1
> ```

Pulling patch on latest master seems to pass validations for me

File: messaging-experiments.yaml [hunk 1]
```diff
@@ -200,3 +200,118 @@
                   label: Start Browsing
                   action:
                     navigate: true
+- id: MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST
+  enabled: true
+  # The filter `!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)` is used to exclude
+  # users who have switched channels or re-installed the browser without having a new profile
+  targeting: env.version >= '81.' && env.channel == 'default' && localeLanguageCode == "en" && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'zero-stage-test' in activeExperiments)
```
Comment (punamdahiya) [inline messaging-experiments.yaml:207]: As discussed, should be updated to use targeting filters for version and channel

File: messaging-experiments.yaml [hunk 2]
```diff
@@ -200,3 +200,118 @@
...
+  arguments:
+    slug: zero-stage-test
```
Comment (punamdahiya) [inline messaging-experiments.yaml:209]: Should be updated to use slug from experimenter, might be worth filing experimenter ticket now

File: messaging-experiments.yaml [hunk 3]
```diff
@@ -200,3 +200,118 @@
...
+    active: true
+    features: []
+    startDate: ""
+    endDate: ""
+    proposedDuration: 0
+    proposedEnrollment: 0
+    referenceBranch: ""
+    bucketConfig:
+      count: 9000
```
Comment (punamdahiya) [inline messaging-experiments.yaml:218]: Assuming we are going to target 20% of population, above number should be changed to 2000

File: messaging-experiments.yaml [hunk 4]
```diff
@@ -200,3 +200,118 @@
...
+      start: 0
+      total: 10000
+      namespace: zero-stage-test
+      randomizationUnit: normandy_id
+    userFacingName: "Multi-Stage About:Welcome Zero Stage"
+    userFacingDescription: Test of the Zero Stage JSON flag, with and without override content
```
Comment (punamdahiya) [inline messaging-experiments.yaml:224]: Should update description to "Study to validate with a minimal onboarding experience."

File: messaging-experiments.yaml [hunk 5]
```diff
@@ -200,3 +200,118 @@
...
+                      navigate: true
+                  secondary_button:
+                    text: Have an account?
+                    label: Sign in
+                    position: top
+                    action:
+                      type: OPEN_URL
+                      addFlowParams: true
+                      data:
+                        args: https://accounts.firefox.com/?service=sync&action=email&context=fx_desktop_v3&entrypoint=activity-stream-firstrun&style=trailhead
```
Comment (punamdahiya) [inline messaging-experiments.yaml:266]: this looks old JSON as we no longer uses args

https://searchfox.org/mozilla-central/source/browser/components/newtab/content-src/lib/aboutwelcome-utils.js#88

File: messaging-experiments.yaml [hunk 6]
```diff
@@ -200,3 +200,118 @@
...
+                        tooltip:
+                          string_id: onboarding-multistage-theme-tooltip-automatic
+                      - theme: light
+                        label: Light
+                        tooltip:
+                          string_id: onboarding-multistage-theme-tooltip-light
+                      - theme: dark
+                        label: Dark
+                        tooltip:
+                          string_id: onboarding-multistage-theme-tooltip-dark
```
Comment (punamdahiya) [inline messaging-experiments.yaml:308]: Its missing Alpenglow, will be good to compare JSON with https://searchfox.org/mozilla-central/source/browser/components/newtab/content-src/lib/aboutwelcome-utils.js

**Pull request closed without merging**